Docs/Configuration.tex

```diff
@@ -2400,6 +2400,17 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
 
   \emph{Note}: This option should be avoided whenever possible. Modern firmware
   usually have compatible AHCI controllers.
+  
+  \item
```

Comment:
The indentation is a bit different compared to the upper item, please fix, since other changes are needed anyway.

Reply:
done